src/data/hold_effects.h

```diff
@@ -0,0 +1,725 @@
+const struct HoldEffectInfo gHoldEffectsInfo[HOLD_EFFECT_COUNT] =
+{
+    [HOLD_EFFECT_NONE] =
+    {
+
+    },
```

Comment:
Not sure if I'm a fan of having some bits just be "empty" like this, but maybe that's just because the extra newline between makes them take so much space?